src/main.ts

```diff
@@ -90,6 +90,7 @@ async function run(): Promise<void> {
       setOutput('credentials_file_path', credentialsPath);
       exportVariable('CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE', credentialsPath);
       exportVariable('GOOGLE_APPLICATION_CREDENTIALS', credentialsPath);
+      exportVariable('GOOGLE_GHA_CREDS_PATH', credentialsPath);
```

Comment:
@sethvargo I added a `_PATH` suffix from what we discussed since I felt that was more representative of the value

Reply:
Is this for users to access in later steps?

Reply:
IIRC `GOOGLE_APPLICATION_CREDENTIALS` does not auth gcloud so I thought it would be confusing if we required that for `setupgcloudsdk`. Using `GOOGLE_GHA_CREDS_PATH` it is also unlikely that the user will configure this manually, so we can give a more actionable error like use `auth` action.

Reply:
I think we need to detect _all_ of these to maximize compat fwiw.

Reply:
@sethvargo I was thinking if we restrict to just `GOOGLE_GHA_CREDS_PATH`, we can nudge them to use auth and hopefully WIF as opposed to exporting a key and setting it as `GOOGLE_APPLICATION_CREDENTIALS` env var within the workflow.